backend/src/schema/resolvers/login.js

```diff
@@ -1,42 +1,60 @@
 const { AuthenticationError } = require("apollo-server-errors");
-const jwt = require("jsonwebtoken");
-
-import { userData } from "../../data";
+import jwt from "jsonwebtoken";
 
 export const LoginResolver = {
   Mutation: {
-    login: (object, params) => {
-      let user = userData.find(user => user.username === params.username);
+    login: async (object, params, ctx) => {
+      const session = ctx.driver.session();
+      try {
+        const result = await session.run(
+          "MATCH (max:User) WHERE max.name = $username return max as user",
```

Comment:
```suggestion
          "MATCH (user:User) WHERE user.name = $username RETURN user {.*}",
```
Then you can skip the call of `.properties()` on line 16 :point_down: